src/makeRobot.js

```diff
@@ -38,6 +38,53 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
+  const Robot = {
```

Comment:
```suggestion
  const robot = {
```
по-моему, не принято писать названние переменной с заглавной буквы